### DISCUSSION TITLE
Start song from start when dj disconnects

### DISCUSSION THREAD

**Discussion by milanwdewaele** [AzuraCast/AzuraCast#3946] [Q&A]
Hello! Ive been using AzuraCast for a while.
But I was thinking. Is it possible for the autodj to start a song from the beginning when a dj disconnects?
If so, could someone help me with that?

Thanks!

**Comment by DrakeAedus** [comment]
This has been bothering me for a long time, and it wasn't an issue when I was using spacial's hosting (samcloud).

Anyone have a solution? I would just like AutoDJ to purge its current song when a live DJ connects, and return with a new song when they disconnect.

**Comment by BusterNeece** [comment]
This is a very long-standing issue we've had with the Liquidsoap AutoDJ tool, and we expect that this will be fixed in version 2.0 of their software, which we're currently in the process of testing.

**Reply by DrakeAedus** [reply]
That sounds a lot better than me writing a webhook to have an API Key auth'd page send a skip command to our back-end when the DJs disconnect.

(I also suspected that I would not be able to simply override the definition for live_disconnected() later in the configs as a user...)

**Reply by BusterNeece** [reply]
Once we're able to get the "skip song after a DJ disconnects" functionality implemented in the next Liquidsoap version, it will be the default functionality, as a vast majority of stations expect things to operate that way.
